src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -370,6 +370,27 @@ def test_aks_create_with_openservicemesh_addon(self, resource_group, resource_gr
             self.check('addonProfiles.openServiceMesh.enabled', True),
         ])
 
+    @AllowLargeResponse()
+    @AKSCustomResourceGroupPreparer(random_name_length=17, name_prefix='clitest', location='westus2')
+    def test_aks_addon_list(self, resource_group, resource_group_location):
+        print("what up?")
+        aks_name = self.create_random_name('cliakstest', 16)
+        self.kwargs.update({
+            'resource_group': resource_group,
+            'name': aks_name,
+        })
+
+        create_cmd = 'aks create --resource-group={resource_group} --name={name} --enable-managed-identity --generate-ssh-keys -o json'
```

Comment:
Please replace `--generate-ssh-keys` with `--ssh-key-value`, this could avoid race condition during concurrent testing. Refer issue [#18854](https://github.com/Azure/azure-cli/issues/18854).